### PR TITLE
Fixed image classification infer.py, do not optimize out

### PR DESCRIPTION
**Pull request by shippingwang**
1. use memory optimizer will cover output memory, add a skip option.

### CODE REVIEW

File: fluid/PaddleCV/image_classification/infer.py [hunk 1]
```diff
@@ -7,11 +7,11 @@
 import sys
 import paddle
 import paddle.fluid as fluid
-import models
+import models_name as models
 import reader
 import argparse
 import functools
-from models.learning_rate import cosine_decay
+from utils.learning_rate import cosine_decay
```
Comment (qingqing01) [inline fluid/PaddleCV/image_classification/infer.py]: remove this line. Infrence does not need lr.

File: fluid/PaddleCV/image_classification/infer.py [hunk 2]
```diff
@@ -7,11 +7,11 @@
 import sys
 import paddle
 import paddle.fluid as fluid
-import models
+import models_name as models
```
Comment (qingqing01) [inline fluid/PaddleCV/image_classification/infer.py]: Also need models?

File: fluid/PaddleCV/image_classification/infer.py (hunk 1 above)
Comment (shippingwang) [reply fluid/PaddleCV/image_classification/infer.py]: OK

File: fluid/PaddleCV/image_classification/infer.py (hunk 2 above)
Comment (shippingwang) [reply fluid/PaddleCV/image_classification/infer.py]: OK

File: fluid/PaddleCV/image_classification/infer.py [hunk 3]
```diff
@@ -11,7 +11,6 @@
 import reader
 import argparse
 import functools
-from models.learning_rate import cosine_decay
```
Comment (jerrywgz) [inline fluid/PaddleCV/image_classification/infer.py]: 这里cosine decay删掉了？

Comment (shippingwang) [reply fluid/PaddleCV/image_classification/infer.py]: > 这里cosine decay删掉了？

Yes

**Review by jerrywgz — COMMENTED** [review]
LGTM

**Review by qingqing01 — APPROVED** [review]
> Fix IC infer bug

Please note PR title...

**Pull request merged**